Extension/src/LanguageServer/settings.ts

```diff
@@ -142,6 +142,7 @@ export interface SettingsParams {
     maxConcurrentThreads: number | null | undefined;
     maxCachedProcesses: number | null | undefined;
     maxMemory: number | null | undefined;
+    maxSymbolSearchResults: number | null | undefined;
```

Comment:
Is this supposed to be null?

Reply:
no. @fearthecowboy, can you fix this one too?